R/Functions_L1_Pop.R

```diff
@@ -1,4 +1,12 @@
 # ---- Level 1 Pop Functions  ----
+utils::globalVariables("colony")
```

Comment:
These need to be here because of CRAN. These variables are used within foreach and then the check complains they are not defined anywhere